### PR TITLE
6d3eda

### PR DESCRIPTION
**Pull request by Koeng101**
Few issues here:

1. Need category info
2. There are 2 labwares that were asked for in the original protocol. In the new labwares library, there are several new options, like offsets. @ncdiehl11 reached out to Luca to get this information. These labwares must be added before pushing.
2. Tip tracking may be useful in this protocol. This still would need to be added, if they don't on just doing full plates. @ncdiehl11 could we get a link in this pull request for me to look at to implement this feature?

### CODE REVIEW

File: protocols/6d3eda/6d3eda.ot2.apiv2.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        sample_plates.append(ctx.load_labware(input_labware, '3'))
+        target_plates.append(ctx.load_labware(output_labware, '6'))
+
+    # Variable pipettes?
+    p300s = ctx.load_instrument(
+            'p300_single_gen2', "right", tip_racks=tip_racks)
+
+    for i, sample_plate in enumerate(sample_plates):
+        for well_num, well in enumerate(sample_plate.wells()[:count-(i*24)]):
+            p300s.transfer(volume, well, target_plates[i].wells()[well_num],
```
Comment (chazmanc) [inline protocols/6d3eda/6d3eda.ot2.apiv2.py:21]: @Koeng101 if you'd like to add tip tracking, a simple version might look like this:

> try:
>      p300s.pick_up_tip()
> except protocol_api.labware.OutOfTipsError:
>     protocol.pause("Replace the tips")
>     p300s.reset_tipracks()
>     p300s.pick_up_tip()

If you put this in the for loop before `p300s.transfer(...` and remove the `new_tip='always')`, you should be good

Comment (Koeng101) [reply protocols/6d3eda/6d3eda.ot2.apiv2.py:27]: Added with commit f792070

Comment (chazmanc) [reply protocols/6d3eda/6d3eda.ot2.apiv2.py:27]: One thing to note, about this is that the proocol_api in the `protocol_api.labware.OutOfTipsError`: should be imported at the top of the file

Comment (chazmanc) [reply protocols/6d3eda/6d3eda.ot2.apiv2.py:27]: For example, [see this](https://github.com/Opentrons/Protocols/blob/develop/protocols/21e4d8-pt2/21e4d8-pt2.ot2.apiv2.py)

**Pull request merged**